src/components/BrowserCell/BrowserCell.react.js

```diff
@@ -304,6 +305,30 @@ export default class BrowserCell extends Component {
     return contextMenuOptions;
   }
 
+  async executeSript(script) {
+      try {
+        const object = Parse.Object.extend(
+          this.props.className
+        ).createWithoutData(this.props.objectId);
+        const response = await Parse.Cloud.run(
+          script.cloudCodeFunction,
+          { object: object.toPointer() },
+          { useMasterKey: true }
+        );
+        this.props.showNote(
+          response || `${script.title} ran with object ${object.id}}`
```

Comment:
Could you add the class name here? I know, not part of this PR, but since you're already rewriting this.

```
`Ran script "${script.title}" on "${this.props.className}" object "${object.id}".`
```

I think there was a also a curly bracket too much at the end?

Reply:
fixed it will push with other changes